compiler/luci/pass/src/PropagateConcatenationQparam.test.cpp

```diff
@@ -48,19 +48,32 @@ int32_t quantize(float f, luci::CircleQuantParam *qparam)
 class SimpleConcatGraph
 {
 public:
-  SimpleConcatGraph()
+  SimpleConcatGraph(loco::DataType quant_type)
   {
-    concat_node.dtype(loco::DataType::U8);
+    concat_node.dtype(quant_type);
     concat_node.fusedActivationFunction(luci::FusedActFunc::NONE);
-    input_1.dtype(loco::DataType::U8);
-    input_2.dtype(loco::DataType::U8);
+    input_1.dtype(quant_type);
+    input_2.dtype(quant_type);
 
     concat_node.values(0, &input_1);
     concat_node.values(1, &input_2);
 
-    addQuantParam(concat_node, {3.14}, {77});
-    addQuantParam(input_1, {1.0}, {1});
-    addQuantParam(input_2, {2.0}, {2});
+    if (quant_type == loco::DataType::U8)
+    {
+      addQuantParam(concat_node, {3.14}, {77});
+      addQuantParam(input_1, {1.0}, {1});
+      addQuantParam(input_2, {2.0}, {2});
+    }
+    else if (quant_type == loco::DataType::S16)
+    {
+      addQuantParam(concat_node, {3.14}, {0});
+      addQuantParam(input_1, {1.0}, {0});
+      addQuantParam(input_2, {2.0}, {0});
+    }
+    else
+    {
+      throw std::runtime_error("Wrong quantization type");
```

Comment:
How about using `Invalid` or `Unsupported` instead of `Wrong`?

Reply:
Updated